### ISSUE TITLE
Null connection returned

### ISSUE THREAD

**Issue by Lakston** [serge-rider/dbeaver#2406]
I can't connect to a postgreSQL database with DBeaver on osx (latest) and get this cryptic `null connection returned`
I can however connect with teamSQL without a problem so my connection infos are the right ones. I checked a ton of resources but I have no idea of the cause of this problem.

For information I have java installed :

```
java -version java version "9.0.1" 
Java(TM) SE Runtime Environment (build 9.0.1+11) 
Java HotSpot(TM) 64-Bit Server VM (build 9.0.1+11, mixed mode)
```

**Comment by kyleflan** [comment]
I already answered on StackOverflow, but make sure you're using the latest version of DBeaver that supports Java 9, e.g. [4.2.3](https://dbeaver.jkiss.org/2017/10/15/dbeaver-4-2-3/).

If you're using the latest and still can't connect, please attach your [logs](https://github.com/serge-rider/dbeaver/wiki/Log-files).

**Comment by Lakston** [comment]
I'm using version 4.2.3

Thanks for your time ! You pointed me to the logs (didn't know where they were) and it helped me resolve the problem, I had a `/` at the end of my url so the connection was ending in `.com/:1244` and same was for the start of it, I needed to strip the `http://`. My colleagues are on windows and didn't have to do that though and the same connection credentials wee working on teamSQL too.

So it looks like DBeaver is stripping those unwanted characters on windows but not on OSX.

Closing the issue, hopefully it will help someone in the future !
